activesupport/lib/active_support.rb

```diff
@@ -41,6 +41,7 @@ module ActiveSupport
   autoload :ExecutionWrapper
   autoload :Executor
   autoload :FileUpdateChecker
+  autoload :ForkTracker
```

Comment:
Sorry.. can you please push this down one more line? `EventedFileUpdateChecker` is out of order because it goes with `FileUpdateChecker`, so best we don't split them.

Otherwise ✅ 🚀 :shipit:

Reply:
@matthewd done!